core/src/main/kotlin/com/bumble/appyx/debug/Appyx.kt

```diff
@@ -1,16 +1,9 @@
 package com.bumble.appyx.debug
 
-object Appyx {
+import com.bumble.appyx.Appyx
 
-    var exceptionHandler: ((Exception) -> Unit)? = null
-
-    fun reportException(exception: Exception) {
-        val handler = exceptionHandler
-        if (handler != null) {
-            handler(exception)
-        } else {
-            throw exception
-        }
-    }
-
-}
+@Deprecated(
+    message = "Moved to com.bumble.appyx package",
+    replaceWith = ReplaceWith("Appyx", imports = ["com.bumble.appyx.Appyx"]),
+)
+typealias Appyx = Appyx
```

Comment:
At what point do we remove this? 1.0.0? 1.1.0?

Reply:
Do not have any plans, impact is miserable.